### ISSUE TITLE
Please merge description of TRLE protocol from existing sources.

### ISSUE THREAD

**Issue by tomuo** [rfbproto/rfbproto#10]
The documentation for the TRLE protocol is MIA.

It was originally proposed in this 2009 posting to tigervnc-rfbproto mailing list:
https://www.mail-archive.com/tigervnc-rfbproto@lists.sourceforge.net/msg00235.html

and it already occurs in the RFC:
https://tools.ietf.org/html/rfc6143#section-7.7.5

**Comment by Neustradamus** [comment]
@tomuo, @CendioOssman: Any news on it?

**Comment by CendioOssman** [comment]
No, I don't think anyone has had a look at it. But given that there is the RFC to look at, and the existing ZRLE documentation, this should not be too hard to add.

Is any implementation using TRLE though?

**Comment by peda-r** [comment]
I have a client implementation here:

https://sourceforge.net/p/ggivnc/code/ci/master/tree/src/encoding/trle.c

And a server implementation here:

https://github.com/antrik/libggi/blob/master/display/vnc/trle.c
